ChangeLog

```diff
@@ -41,7 +41,11 @@ ver. 0.9.2 (2014/XX/XXX) - wanna-be-released
    - Monit config for fail2ban in /files/monit
    - New actions:
      - action.d/firewallcmd-multiport and action.d/firewallcmd-allports Thanks Donald Yandt
-     
+   - New status commands:
+     - fail2ban-client status <jail> extended
+       - prints Cymru data (ASN, Country RIR) per banned IP
+       - Requires dnspython or dnspython3
```

Comment:
Agreed, I thought there was a better place to add dependency info :)  8fd9d6a770bdfc90d59da6c1493d82e0b1a2752e